### ISSUE TITLE
Latest version of Docker image engine-jupyter missing bblfsh

### ISSUE THREAD

**Issue by campoy** [src-d/engine#405]
## Expected Behavior
Running engine-jupyter should allow me to run the example notebook included.

## Current Behavior
Currently running it fails on the first line due to missing bblfsh.

## Possible Solution
In order to have it working again I had to run the following in the container:

```
apt-get install curl g++ libxml2-dev && pip install bblfsh
```

## Steps to Reproduce (for bugs)
1. sudo docker run -d --name engine-jupyter -p 8080:8080 -v $(pwd)/repos
itories:/repositories --link bblfshd:bblfshd srcd/engine-jupyter
2. open Example notebook
3. run first code snippet

## Context
Trying to demo the engine.

## Your Environment (for bugs)

Running on Docker from Debian instance.

```
$ sudo docker version
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:17:14 2018
 OS/Arch:      linux/amd64
 Experimental: false
 Orchestrator: swarm
Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:15:24 2018
  OS/Arch:      linux/amd64
  Experimental: false
```

**Comment by ajnavarro** [comment]
Bblfsh is in a separated container. Explained here: https://github.com/src-d/engine#playing-around-with-engine-on-jupyter

**Comment by campoy** [comment]
This is not acceptable.

Bblfsh running on a different container is totally fine, but the `bblfsh` package not being installed by default with the `engine`, taking into account that creating a new instance will fail, is simply not ok.

This is what the experience looks like (even running `bblfsh` on a linked container right now)

```
9:56 ~ ➜  docker run -d --name bblfshd --privileged -v storage:/var/lib/bblfshd bblfsh/bblfshd
1b9311f7657895171b5634a510032f63a9cb0db7f6932af47809d02e74b92a18
9:56 ~ ➜  docker run --name engine-jupyter --rm -it -p 8080:8080 -v /Users/francesc/siva:/repositories --link bblfshd:bblfshd srcd/engine-jupyter
[I 16:57:10.238 NotebookApp] Writing notebook server cookie secret to /root/.local/share/jupyter/runtime/notebook_cookie_secret
[W 16:57:10.544 NotebookApp] All authentication is disabled.  Anyone who can connect to this server will be able to run code.
[I 16:57:10.591 NotebookApp] Jupyter-Spark enabled!
[I 16:57:10.595 NotebookApp] Serving notebooks from local directory: /home
[I 16:57:10.595 NotebookApp] 0 active kernels
[I 16:57:10.595 NotebookApp] The Jupyter Notebook is running at:
[I 16:57:10.596 NotebookApp] http://0.0.0.0:8080/
[I 16:57:10.596 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 16:57:14.509 NotebookApp] 302 GET / (172.17.0.1) 0.93ms
[I 16:57:19.053 NotebookApp] Writing notebook-signing key to /root/.local/share/jupyter/notebook_secret
[W 16:57:19.056 NotebookApp] Notebook Example.ipynb is not trusted
[I 16:57:19.881 NotebookApp] Kernel started: e8ac66f1-5bbc-424c-80cd-b29aab8651e3
[I 16:57:20.603 NotebookApp] Adapting to protocol v5.1 for kernel e8ac66f1-5bbc-424c-80cd-b29aab8651e3
```
<img width="763" alt="screen shot 2018-06-20 at 9 59 35 am" src="https://user-images.githubusercontent.com/2237452/41673108-ab5a7c40-7470-11e8-869d-14686024f779.png">

Please add that dependency back in.

**Comment by ajnavarro** [comment]
Got it. 

I understood that the missing part was bblfsh itself, not the client library.

**Comment by ajnavarro** [comment]
This fix will be released on engine v0.7.0

Right now you can get the latest generated docker image from master doing:
```
docker pull srcd/engine-jupyter:dev-9b1c91a-dirty
```

**Comment by alejandrosame** [comment]
That works for me. Thanks!
